doc/nrf/libraries/networking/cloud.rst

```diff
@@ -7,6 +7,10 @@ Cloud API
    :local:
    :depth: 2
 
+.. note::
+
+   The cloud API library is deprecated. Use the desired cloud service library directly.
```

Comment:
Deprecation means that an API will be removed in the future, but customers can keep using it. However this change removes it without prior deprecation.

It seems to me that this change actually removes the cloud API?

Reply:
the cloud api library remains, but implementations of it have been removed.